config/permission.php

```diff
@@ -77,13 +77,4 @@
      */
 
     'cache_expiration_time' => 60 * 24,
-
-    /*
-     * By default we'll make an entry in the application log when the permissions
-     * could not be loaded. Normally this only occurs while installing the packages.
-     *
-     * If for some reason you want to disable that logging, set this value to false.
-     */
-
-    'log_registration_exception' => true,
```

Comment:
With this removed, can we also remove `shouldLogException()` from `PermissionRegistrar.php` ?

Reply:
Yep, I've removed it now 👍